iati/core/tests/test_rulesets.py

```diff
@@ -187,14 +183,38 @@ class RuleSubclassTestBase(object):
     """A base class for Rule subclass tests."""
 
     @pytest.fixture
-    def basic_rule(self, rule_constructor, valid_case):
+    def xpath_base(self):
+        """Return valid xpath_base."""
+        return '//root_element'
+
+    @pytest.fixture
+    def empty_xpath_base(self):
+        """Empty_xpath_base."""
+        return ''
+
+    @pytest.fixture
+    def basic_rule(self, rule_constructor, valid_case, xpath_base):
```

Comment:
Other fixtures are named `rule_[xxx]` - this is inconsistent and should be brought in line with the others.